### ISSUE TITLE
"Ads notifications received this month" should only increment for views with a redemption value greater than 0.0

### ISSUE THREAD

**Issue by tmancey** [brave/brave-browser#10020]
## Description 

"Ads notifications received this month" should only increment for views with a redemption value greater than 0.0

## Steps to Reproduce

   1. Click or dismiss an ad that has a redemption value for a click or dismiss greater than 0.0 (will need adding server-side)

## Actual result:

"Ad notifications received this month" is incremented

## Expected result:
"Ad notifications received this month" should not increment

## Version/Channel Information:

- Can you reproduce this issue with the current release?
- Can you reproduce this issue with the beta channel?
- Can you reproduce this issue with the dev channel?
- Can you reproduce this issue with the nightly channel?

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome?

**Comment by GeetaSarvadnya** [comment]
Blocked until the PR https://github.com/brave/brave-core/pull/5751 is uplifted
for more info https://bravesoftware.slack.com/archives/C43LW5U2H/p1592560782311300?thread_ts=1592560491.310900&cid=C43LW5U2H

**Comment by GeetaSarvadnya** [comment]
PR #5751 is uplifted to 1.11.x hence removing QA blocked label

**Comment by LaurenWags** [comment]
Added `QA/Blocked` label - need assistance on this one - https://bravesoftware.slack.com/archives/C43LW5U2H/p1593628267348700?thread_ts=1593173388.321300&cid=C43LW5U2H

**Comment by LaurenWags** [comment]
Verified passed with

```
Brave | 1.11.85 Chromium: 83.0.4103.116 (Official Build) dev (64-bit)
-- | --
Revision | 8f0c18b4dca9b6699eb629be0f51810c24fb6428-refs/branch-heads/4103@{#716}
OS | macOS Version 10.14.6 (Build 18G3020)
```

* Verified STR from description using [non-zero-catalog.txt](https://github.com/brave/brave-browser/files/4866431/non-zero-catalog.txt)

* Confirmed `view` (with estimated earnings of 0.3 BAT) was counted in my `Ad notifications received this month` but the `dismiss` (also worth estimated 0.3 BAT) was not included:

```
[20095:775:0702/160854.342957:VERBOSE1:redeem_token.cc(376)] Added 1 unblinded payment token with an estimated redemption value of 0.3 BAT, you now have 1 unblinded payment tokens
[20095:775:0702/160854.343014:VERBOSE3:confirmations_impl.cc(748)] Saving confirmations state
[20095:775:0702/160854.343582:VERBOSE1:redeem_token.cc(404)] Successfully redeemed token with confirmation id 83cb9575-a280-4c05-ba8e-d769c31e6eae, creative instance id 68bc0caf-04d2-425a-8865-1acd6cbb63f1 and view
```

```
[20095:775:0702/160900.125296:VERBOSE1:redeem_token.cc(376)] Added 1 unblinded payment token with an estimated redemption value of 0.3 BAT, you now have 2 unblinded payment tokens
[20095:775:0702/160900.125348:VERBOSE3:confirmations_impl.cc(748)] Saving confirmations state
[20095:775:0702/160900.126397:VERBOSE1:redeem_token.cc(404)] Successfully redeemed token with confirmation id b453d099-37d6-459d-8f2a-f990d70db94e, creative instance id 68bc0caf-04d2-425a-8865-1acd6cbb63f1 and dismiss
```

<img width="651" alt="Screen Shot 2020-07-02 at 4 12 06 PM" src="https://user-images.githubusercontent.com/28145373/86405169-edd0b280-bc7e-11ea-8581-78d6d7ace6bf.png">
<img width="312" alt="Screen Shot 2020-07-02 at 4 13 26 PM" src="https://user-images.githubusercontent.com/28145373/86405207-fb863800-bc7e-11ea-933c-caa872387d32.png">

* Confirmed `view` (with estimated earnings of 0.3 BAT) was counted in my `Ad notifications received this month` but the `click` and `landed` (also each worth estimated 0.3 BAT) were not included:

```
[20154:775:0702/161720.157914:VERBOSE1:redeem_token.cc(376)] Added 1 unblinded payment token with an estimated redemption value of 0.3 BAT, you now have 1 unblinded payment tokens
[20154:775:0702/161720.157971:VERBOSE3:confirmations_impl.cc(748)] Saving confirmations state
[20154:775:0702/161720.158499:VERBOSE1:redeem_token.cc(404)] Successfully redeemed token with confirmation id f93a5910-ece8-4d25-8d61-509dd1375e55, creative instance id 68bc0caf-04d2-425a-8865-1acd6cbb63f1 and view
```

```
[20154:775:0702/161725.332128:VERBOSE1:redeem_token.cc(376)] Added 1 unblinded payment token with an estimated redemption value of 0.3 BAT, you now have 2 unblinded payment tokens
[20154:775:0702/161725.332183:VERBOSE3:confirmations_impl.cc(748)] Saving confirmations state
[20154:775:0702/161725.332714:VERBOSE1:redeem_token.cc(404)] Successfully redeemed token with confirmation id adf5f36f-df43-41d9-9ca2-0cf903fe4863, creative instance id 68bc0caf-04d2-425a-8865-1acd6cbb63f1 and click
```

```
[20154:775:0702/161737.441432:VERBOSE1:redeem_token.cc(376)] Added 1 unblinded payment token with an estimated redemption value of 0.3 BAT, you now have 3 unblinded payment tokens
[20154:775:0702/161737.441486:VERBOSE3:confirmations_impl.cc(748)] Saving confirmations state
[20154:775:0702/161737.441988:VERBOSE1:redeem_token.cc(404)] Successfully redeemed token with confirmation id 3e95d86f-92e8-4d5d-b541-f9f9563c40c8, creative instance id 68bc0caf-04d2-425a-8865-1acd6cbb63f1 and landed
```

<img width="655" alt="Screen Shot 2020-07-02 at 4 19 06 PM" src="https://user-images.githubusercontent.com/28145373/86405711-ccbc9180-bc7f-11ea-8def-e6c5096c4987.png">
<img width="304" alt="Screen Shot 2020-07-02 at 4 19 16 PM" src="https://user-images.githubusercontent.com/28145373/86405716-cf1eeb80-bc7f-11ea-8a56-59ab02fde62b.png">
